src/posts/2020-06-08-going-remote.md

```diff
@@ -0,0 +1,110 @@
+
+A quick workaround this that I've seen many teams implement is asking "hey, remote folks, what are your thoughts?". The problem here is the timing. This usually happens after the topic has already been discussed, as to not break the conversational flow. Quite frankly, most of the time yes, you wanted to aadd something but now people have already discussed what they wanted and what you wanted to say was referring to something several minutes earlier. Now everyone is looking at the screen waiting for you to say something and you have two options. Either 1. you decide to go ahead, speak your mind and rewind the conversation, say what you wanted which will make the conversation loop and is very commonly followed by a "I think we're looping and we should probably discuss this on Slack/Basecamp/X" (to be fair quite rightfully so, you are indeed looping). Or 2. you do understand the awkwardness of forcing the conversation to loop, feel that the train has already left the station and just go with *"nah, I'm good"*, which in time will feed the isolation and powerlessness you feel.
+
+I was (probably) the first one to notice this pattern in our company and addressed it. After dissecting we concluded **there is no easy solution but to make a conscious effort** of noticing when someone remote is trying to talk. **If you're in the office, you need to be the champion of this**. You need to take charge and say "hold on, I think \<Lovely Person Who Happens To Be Remote\> wants to add something". You need to place yourself in the shoes (slippers, probably) of the people who are remote. **It is also your responsibility, as an awesome coworker, to make sure the people who are not in office feel comfortable, empowered, and you're all having a pleasant experience**.
+
+### In Summary
+
+So to recap, these are our steps for an awesome remote culture. They might not apply to you or your problems. But they worked out pretty well for us:
+
+* Create moments for goofing around.
+* Dailies are not those moments.
+* Chat with people, most of them have something interesting to say, like the fact that the Sicilian defence is overrated.
+* Send beer from your beer fountain to coworkers.
+* Having meetings with people in office is not a sin, but not communicating decisions and outcomes will bring you 36514 years of bad luck.
+* Use tools like Basecamp to communicate asynchronously.
+* Don't do too many meetings.
+* When you do meetings, if you're in office stand by your remote coworkers and make sure they get a timely chance to speak their minds.
+* And finally: implement Margarita Tuesdays to the sound of Cardi B. If you're not allowed, implement them by yourself at home because you now have an awesome remote culture.
+
+Cheers,
+
+Mendes
```

Comment:
Use a line break (`<br>`) instead of forcing another paragraph, otherwise the spacing will look very weird.

```suggestion
Cheers,<br>Mendes
```